### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by kokodavid**
My profile page widget tree fixes and event page fixes

### CODE REVIEW

**Comment by kokodavid** [comment]
image
image

File: .changes/0832-eventpage-fixes.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- [fix] fixed My profile page widget tree errors 
+- [fix] fixed Events page card display errors
```
Comment (gnunicorn) [inline .changes/0832-eventpage-fixes.md:2]: Remember the changelog entries are meant for _end_ users to understand. They don't understand what a "widget tree error" is ...

```suggestion
- [fix] Cases in which 'My profile' showed up blank 
- [fix] 'Events page' showing errors for specific events. 
```

File: app/lib/features/profile/pages/my_profile_page.dart [hunk 2]
```diff
@@ -176,100 +174,88 @@ class MyProfile extends ConsumerWidget {
...
-                children: <Widget>[
-                  Wrap(
-                    children: [
-                      Container(
-                        margin: const EdgeInsets.fromLTRB(20, 15, 20, 0),
+              Container(
+                margin: const EdgeInsets.fromLTRB(20, 15, 20, 0),
+                decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(25),
+                ),
```
Comment (gnunicorn) [inline app/lib/features/profile/pages/my_profile_page.dart:181]: I wonder if we should outsource this repeating boxdecoration into the theme maybe?

**Review by gnunicorn — APPROVED** [review]
Looking good, just minor remarks that don't have to be addressed in this PR specifically.

File: .changes/0832-eventpage-fixes.md (hunk 1 above)
Comment (kokodavid) [reply .changes/0832-eventpage-fixes.md:2]: Noted ,  I'll be doing that .

File: app/lib/features/profile/pages/my_profile_page.dart [hunk 3]
```diff
@@ -176,100 +174,88 @@ class MyProfile extends ConsumerWidget {
...
-                                        'Username copied to clipboard',
-                                      );
-                                    },
-                                  ),
-                                ],
+                    ),
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        Text(account.profile.displayName ?? ''),
```
[the remark this answers is not in the document]
Comment (kokodavid) [reply app/lib/features/profile/pages/my_profile_page.dart:220]: Yes we can do that .

File: app/lib/features/profile/pages/my_profile_page.dart (hunk 2 above)
Comment (kokodavid) [reply app/lib/features/profile/pages/my_profile_page.dart:181]: Like have it in the App Theme ?

File: app/lib/features/profile/pages/my_profile_page.dart (hunk 3 above)
[the remark this answers is not in the document]
Comment (kokodavid) [reply app/lib/features/profile/pages/my_profile_page.dart:220]: Can we have 'Add Username' displayed in italics if it's null  ?

[the remark this answers is not in the document]
Comment (gnunicorn) [reply app/lib/features/profile/pages/my_profile_page.dart:220]: sure.

File: app/lib/features/profile/pages/my_profile_page.dart (hunk 2 above)
Comment (gnunicorn) [reply app/lib/features/profile/pages/my_profile_page.dart:181]: yeah, like that. probably.

**Pull request merged**